contracts/ColonyFunding.sol

```diff
@@ -132,15 +135,18 @@ contract ColonyFunding is ColonyStorage, PatriciaTreeProofs {
     return fundingPots[_potId].balance[_token];
   }
 
-  function getFundingPot(uint256 _potId) public view returns (FundingPotAssociatedType associatedType, uint256 associatedTypeId) {
-    FundingPot storage pot = fundingPots[_potId];
-    return (pot.associatedType, pot.associatedTypeId);
+  function getFundingPotPayout(uint256 _potId, address _token) public view returns (uint256) {
+    return fundingPots[_potId].payouts[_token];
   }
 
-  function moveFundsBetweenPots(uint256 _fromPot, uint256 _toPot, uint256 _amount, address _token) public
-  stoppable
-  auth
+  function getFundingPot(uint256 _potId) public view returns
+  (FundingPotAssociatedType associatedType, uint256 associatedTypeId, uint256 payoutsWeCannotMake)
   {
+    FundingPot storage fundingPot = fundingPots[_potId];
+    return (fundingPot.associatedType, fundingPot.associatedTypeId, fundingPot.payoutsWeCannotMake);
+  }
+
+  modifier test(uint256 _fromPot, uint256 _toPot) {
```

Comment:
Note to self: Name this properly and move to where modifiers live in the `DataTypes` contract